components/supervisor/pkg/metrics/reporter.go

```diff
@@ -41,9 +41,8 @@ func NewGrpcMetricsReporter(gitpodHost string) *GrpcMetricsReporter {
 			"grpc_server_handling_seconds":        true,
 			"supervisor_ide_ready_duration_total": true,
 			"supervisor_initializer_bytes_second": true,
-			"grpc_client_started_total":           true,
-			"grpc_client_handled_total":           true,
-			"grpc_client_handling_seconds":        true,
+			"supervisor_client_handled_total":     true,
+			"supervisor_client_handling_seconds":  true,
```

Comment:
Was thinking yesterday, why do we need this other check here too? we already filter metrics in ide-metrics, can't we just remove this `supportedMetrics` map? :thinking: cc @iQQBot @mustard-mh

Reply:
Maybe one way to save traffic